src/System.Runtime.Extensions/tests/System/IO/Path.Combine.cs

```diff
@@ -154,25 +161,60 @@ public static void ContainsInvalidCharWithoutRootedAfterArgumentNull_Windows()
             CommonCasesException<ArgumentException>("ab\tcd");
         }
 
-        [ActiveIssue(25665)]
         [Fact]
-        public static void ContainsInvalidCharWithRootedAfterArgumentNull()
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public static void ContainsInvalidCharWithoutRootedAfterArgumentNull_Windows_Core()
+        {
+            Assert.Equal("ab|cd", Path.Combine("ab|cd"));
+            Assert.Equal("ab\bcd", Path.Combine("ab\bcd"));
+            Assert.Equal("ab\0cd", Path.Combine("ab\0cd"));
+            Assert.Equal("ab\tcd", Path.Combine("ab\tcd"));
+        }
+
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework)]
+        public static void ContainsInvalidCharWithRootedAfterArgumentNull_Desktop()
         {
             //any path contains invalid character with rooted after (AE)
             CommonCasesException<ArgumentException>("ab\0cd", s_separator + "abc");
         }
 
-        [ActiveIssue(25665)]
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        public static void ContainsInvalidCharWithRootedAfterArgumentNull_Core()
+        {
+            if (PlatformDetection.IsWindows)
+            {
+                Assert.Equal("\\abc", Path.Combine("ab\0cd", s_separator + "abc"));
+            }
+            else
+            {
+                Assert.Equal("/abc", Path.Combine("ab\0cd", s_separator + "abc"));
+            }
+        }
+
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework)]
         [PlatformSpecific(TestPlatforms.Windows)]  // Tests Windows-specific invalid paths
-        public static void ContainsInvalidCharWithRootedAfterArgumentNull_Windows()
+        public static void ContainsInvalidCharWithRootedAfterArgumentNull_Windows_Desktop()
         {
             //any path contains invalid character with rooted after (AE)
             CommonCasesException<ArgumentException>("ab|cd", s_separator + "abc");
             CommonCasesException<ArgumentException>("ab\bcd", s_separator + "abc");
             CommonCasesException<ArgumentException>("ab\tcd", s_separator + "abc");
         }
 
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        [PlatformSpecific(TestPlatforms.Windows)]  
+        public static void ContainsInvalidCharWithRootedAfterArgumentNull_Windows_Core()
```

Comment:
Same question about "ArgumentNull" in the name.  Will stop asking about this, but the question applies anywhere this occurs and the test isn't doing anythin with that.